bin/crew

```diff
@@ -1122,9 +1138,18 @@ def build_package (pwd)
 end
 
 def archive_package (pwd)
-  pkg_name = "#{@pkg.name}-#{@pkg.version}-chromeos-#{@device[:architecture]}.tar.xz"
-  Dir.chdir CREW_DEST_DIR do
-    system "tar c#{@verbose}Jf #{pwd}/#{pkg_name} *"
+  unless ENV['CREW_USE_PIXZ'] == '1'
```

Comment:
Why do we need an environment variable set to do this?

Reply:
Default should be (for now at least) tar.xz. But especially for large packages, one could set that to generate `.tpxz` archives instead using `pixz`.  If there's a better way to do that...

Reply:
(trying to be conservative here with changes.)

Reply:
Later we can flip to default or not... Let's see compression ratios and how useful tpxz ends up being?

Reply:
You are supposed to be doing that in your testing already. :)

Reply:
This is an inital framework for getting packages stored using tpxz.

I don't want to create bugs by using a standalone crew version for testing... I have to dogfood this, which means using it production with feature flags hiding things. The env variables are our version of feature flags, no?

Reply:
Don't you always create a standalone crew version for testing when you work locally?  It's fine to make this the default if has been fully tested and provides a benefit to improve the compression scheme.

Reply:
Yes I create a local version.... using CREW_TESTING...

We need the option to create two separate archive formats since there's a bootstrapping issue with new installs.